core/controllers/suggestion_test.py

```diff
@@ -320,6 +535,18 @@ def test_suggestion_list_handler(self):
             )['suggestions']
         self.assertEqual(len(suggestions), 2)
 
+    def test_cannot_resubmit_suggestion_with_invalid_suggestion_id(self):
```

Comment:
Can we also test that a user cannot resubmit a suggestion that is not their own? i.e.
1. Create a suggestion as user1
2. Reject it
3. Try resubmitting as user2 and it should fail.

Reply:
The `can_resubmit_suggestion` decorator handle this case and there is a test in acl_decorators_test ResubmitSuggestionDecoratorsTests which handles this

Reply:
Thanks for that... Wasn't sure about that 👍